pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -638,6 +636,19 @@ func (p *AnacondaInstallerISOTree) makeKickstartStages(stageOptions *osbuild.Kic
 
 		}
 		hardcodedKickstartBits += makeKickstartSubscriptionPost(subscriptionPath, systemPath)
+
+		// include a readme file on the ISO in the subscription path to explain what it's for
+		subscriptionReadme, err := fsnode.NewFile(
+			filepath.Join(subscriptionPath, "README"),
+			nil, nil, nil,
+			[]byte(`Subscription services and credentials
```

Comment:
I'm reluctant to suggest this :) But embedding a README's text in the code seems messy. But images really doesn't have a good common place to pull such a file from since it's a library and could end up anywhere.

But a README file could be added to the source tree, and it could be embedded in the build using https://pkg.go.dev/embed, allowing for easy editing of the file as a file.

But maybe I'm overthinking things...

Reply:
Yeah, you're not wrong.  It's quite weird how I added it here.
We do similar things with comments in some config files, but that's a bit different.

Reply:
@bcl Thought about this a bit and I'd like to use embed but I think that should be part of a bigger change that includes a lot of similar changes, like pulling out hardcoded kickstart blobs.

If you don't mind, can we merge this as-is and make a PR with embedding as a follow-up?